src/compositions/Timeline/Timeline.tsx

```diff
@@ -13,7 +13,8 @@ const Timeline = (props: any) => {
     <div className="statusTimeline" data-testid="statusTimeline">
       <ul className="statusTimelineUl">
         {statusConfig.map(
-          (config: any, index: React.Key | null | undefined) => {
+          (_config: any, index: React.Key | null | undefined) => {
+            const config = JSON.parse(JSON.stringify(_config));
```

Comment:
I would suggest to remove this edit now, as the original intend was to modify the config with the calculated values. 

I couldn't test this bit, to ensure this copy-by-value (removing copy-by-reference) works as I started getting loading issues. Hence I suggest to not do this update here.

Reply:
Ok! then I should move this update logic outside of the component rendering

Reply:
Done

Reply:
The edit is not clear to me. I've added another comment though., pls check that